components/landing_page.module.css

```diff
@@ -0,0 +1,135 @@
+/* Wrapper for landing page content */
+.body {
+	margin-right: auto;
+	margin-left: auto;
+	max-width: 1440px;
+	padding-right: 100px;
+	padding-left: 100px;	
+}
+
+/* General Column */
+.column {
+	display: flex;
+	flex-direction: column;
+	flex-basis: 100%;
+	flex: 1;
+	justify-content: center;
+	align-items: center;
+  padding-right: 30px;
+  padding-left: 30px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+
+/* General Row */
+.row {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  max-width: 100%;
+  justify-content: center;
+} 
+
+/* Main Text */
+.TCFG {
+  width: 715px;
+  height: 184px;
+  font-family: GujaratiSangamMN;
+  font-size: 72px;
```

Comment:
It can be really good to set font sizes and line heights using `rem` instead of `px`. That way, if the user manually adjusts their font settings to be larger or smaller, then the website can attempt to respect those preferences. If sizes are set using `px`, then it is static and does not adjust for user preferences. This is especially important in an accessibility context. https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units